src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,73 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected TaskType taskType;
```

Comment:
Tasktype is a redundant variable as it is already differentiated by the inheriting of the ToDo, Event and Deadline classes. Perhaps you could use method overriding in the subclasses ?